examples/quickstarts/helidon-quickstart-se/Dockerfile.native

```diff
@@ -15,7 +15,7 @@
 #
 
 # 1st stage, build the app
-FROM helidon/jdk11-graalvm-maven:21.0.0 as build
+FROM helidon/jdk11-graalvm-maven:21.3.0 as build
```

Comment:
20.3.4 does not appear to have Java 8 support (https://github.com/graalvm/graalvm-ce-builds/releases/tag/vm-20.3.4) so it would be 20.3.3.  Or we push ahead with 21...

Reply:
I am not sure if we have a claim anywhere regarding native image support and version of java. If not, let's go with 21.3.0 and use Java 11. But this may be a limiting factor for users...